### PR TITLE
Example: Cu6+ Benchmark CSCS

### PR DESCRIPTION
**Pull request by ax3l**
# Please do NOT merge this PR

Benchmark with CU6+ and 2keV for @HighIander @psychocoderHPC 

Simulates the main interaction part of the foil with
  BSI + ADK + TF

Note: see inline comments, just tested on Hypnos quickly.

### CODE REVIEW

File: examples/WarmCopper/cmakeFlags [hunk 1]
```diff
@@ -29,8 +29,9 @@
 #   - start with zero index
 #   - increase by 1, no gaps
 
-flags[0]="-DCUDA_ARCH=20 -DCMAKE_DISABLE_FIND_PACKAGE_ISAAC=True"
-flags[1]="-DCUDA_ARCH=20 -DCMAKE_DISABLE_FIND_PACKAGE_ISAAC=True -DPARAM_OVERWRITES:LIST=-DPARAM_ENABLE_PUSHER=1;-DPARAM_ENABLE_CURRENT=1"
+flags[0]="-DCUDA_ARCH=37"
```
Comment (ax3l) [inline examples/WarmCopper/cmakeFlags]: change to 60 on CSCS

File: examples/WarmCopper/include/simulation_defines/param/particle.param [hunk 2]
```diff
@@ -38,46 +38,35 @@ namespace particles
      *  unit: none */
     constexpr float_X MIN_WEIGHTING = 10.0;
 
-    constexpr uint32_t TYPICAL_PARTICLES_PER_CELL = 2;
+    constexpr uint32_t TYPICAL_PARTICLES_PER_CELL = 1; //10;
```
Comment (ax3l) [inline examples/WarmCopper/include/simulation_defines/param/particle.param]: change to put more particles on the GPU

File: examples/WarmCopper/include/simulation_defines/param/particle.param [hunk 3]
```diff
@@ -38,46 +38,35 @@ namespace particles
...
-    using Assign200keVDrift = DriftImpl< Drift200keVParam, nvidia::functors::Assign >;
-
     struct TemperatureParam
     {
-        /*Initial temperature
+        /* Initial temperature e-
          *  unit: keV
          */
-        static constexpr float_64 temperature = 0.1;
+        static constexpr float_64 temperature = 2.0;
```
Comment (ax3l) [inline examples/WarmCopper/include/simulation_defines/param/particle.param]: 2 keV to trigger both field an collisional ionization

File: examples/WarmCopper/include/simulation_defines/param/speciesDefinition.param [hunk 4]
```diff
@@ -138,18 +110,23 @@ value_identifier( float_X, ChargeRatioCopper, -29.0 );
...
     densityRatio< DensityRatioCopper >,
-    atomicNumbers< ionization::atomicNumbers::Copper_t >
+    atomicNumbers< ionization::atomicNumbers::Copper_t >,
+    ionizationEnergies< ionization::energies::AU::Copper_t >,
+    effectiveNuclearCharge< ionization::effectiveNuclearCharge::Copper_t >,
+    ionizers<
+        MakeSeq_t<
+            particles::ionization::BSIEffectiveZ< BulkElectrons >,
+            particles::ionization::ADKLinPol< BulkElectrons >,
+            particles::ionization::ThomasFermi< BulkElectrons >
```
Comment (ax3l) [inline examples/WarmCopper/include/simulation_defines/param/speciesDefinition.param]: affected by #2024

File: examples/WarmCopper/submit/1gpu.cfg [hunk 5]
```diff
@@ -36,26 +36,14 @@ TBG_gpu_x=1
 TBG_gpu_y=1
 TBG_gpu_z=1
 
-TBG_gridSize="-g 32 32 32"
-TBG_steps="-s 1000"
+TBG_gridSize="-g 128 256 128"
```
Comment (ax3l) [inline examples/WarmCopper/submit/1gpu.cfg]: increase at will to fill GPU mem

**Comment by psychocoderHPC** [comment]
Note: This example uses the default `file.param` settings this includes the `EnergyDensity` for all species.
Due to the fact that photons has a mass zero it is not allowed to call the method to calculate gamma (this results in a division by zero).

**Comment by ax3l** [comment]
Really? It should not, it's just an energy divided by a number density which is also well defined for photons...

**Pull request closed without merging**